server/src/db/seeders/ReviewsSeeder.js

```diff
@@ -0,0 +1,39 @@
+import { Review } from "../../models/index.js";
+
+class ReviewSeeder {
+    static async seed () {
+        const reviews = [{
+                reviewText: "My dog really likes this park, it is simple but fun for him",
+                rating: 4,
+                userId: 1,
+                dogParkId: 1
```

Comment:
in seeder files we shouldn't assume id values by hard coding because we do not know if those records will actually exist depending on when the seeder file is run, objects are created/ destroyed, etc. Instead, we can query for the data needed from the tables we are trying to associate, and try to find records based on some criteria like the `name`. Then once we have that object from the database, we can access it's specific `id` to use for the new records we want to persist. 

```js
const exampleUser = await User.query().findOne({ email: "user@example.com" })
...
userId: exampleUser.id
```